public/locales/en/translation.json

```
@@ -37,6 +37,32 @@
     "largeFilesWarning": "If you have issues downloading files larger than 1GB, installing <1>IPFS Desktop</1> or <3>IPFS Companion</3> can improve performance.",
     "viewOnGateway": "View on IPFS"
   },
+  "modal": {
+    "how": {
+      "title": "How IPFS Share works",
+      "ariaLabel": "Modal explaining how IPFS Share works",
+      "copy": "IPFS Share is a simple, free way to share files without needing to rely on services run by big corporations. Instead, it uses the peer-to-peer IPFS protocol for sharing directly between devices. IPFS is built around content addressing — keeping track of files based on what they contain, not where they're located — which means that files can't be modified without changing their addresses too. In other words, when you download a file using IPFS, you can be sure that it's exactly the same as the sender intended.",
+      "learnMore": "Learn more about IPFS and how it's changing the internet by visiting the  <1>IPFS website</1>."
+    },
+      "reprovide": {
+      "title": "Re-sharing files you've received",
+      "ariaLabel": "Modal explaining how to share these files with someone else",
+      "copy": "It's easy to share these files with someone else by making your device part of the IPFS network. All you need to do is keep this page open; by doing so, your device becomes a node on the network and makes those files available directly to others. Then, share the link that you received — and be sure to keep this page open until you know your recipient has downloaded the files.",
+      "learnMore": "Curious how this works under the hood? Check out the <1>official IPFS documentation</1> for everything from basic explainers to developer tools."
+    },
+    "privacy": {
+      "title": "Sharing files on IPFS",
+      "ariaLabel": "Modal describing sharing and privacy on IPFS",
+      "copy": "This site uses IPFS, a peer-to-peer protocol for storing and serving data directly between users without relying on cloud storage or other services run by big corporations. Because IPFS is a public network, when you share content using IPFS Share, you're enabling anyone who knows the unique CID in your share link to retrieve it. This means you should avoid sharing sensitive files or copyrighted material.",
```

Should we change "serving" in the first sentence to "sharing" to be better for less techy folks? Serving connotes being a server farm or using weird hardware to me and sharing connotes being a nice human.

---

```suggestion
      "copy": "This site uses IPFS, a peer-to-peer protocol for storing and sharing data directly between users without relying on cloud storage or other services run by big corporations. Because IPFS is a public network, when you share content using IPFS Share, you're enabling anyone who knows the unique CID in your share link to retrieve it. This means you should avoid sharing sensitive files or copyrighted material.",
```
I'd actually mulled that over both ways. Thanks for breaking the tie!